sdk/python/tests/test_registry.py

```diff
@@ -0,0 +1,347 @@
+)
+def test_apply_entity_integration(test_registry):
+    entity = Entity(
+        name="driver_car_id",
+        description="Car driver id",
+        value_type=ValueType.STRING,
+        labels={"team": "matchmaking"},
+    )
+
+    project: StrictStr = "project"
+
+    # Register Entity
+    test_registry.apply_entity(entity, project)
+
+    entities = test_registry.list_entities(project)
+
+    entity = entities[0]
+    assert (
+        len(entities) == 1
+        and entity.name == "driver_car_id"
+        and entity.value_type == ValueType(ValueProto.ValueType.STRING)
+        and entity.description == "Car driver id"
+        and "team" in entity.labels
+        and entity.labels["team"] == "matchmaking"
+    )
+
+    entity = test_registry.get_entity("driver_car_id", project)
+    assert (
+        entity.name == "driver_car_id"
+        and entity.value_type == ValueType(ValueProto.ValueType.STRING)
+        and entity.description == "Car driver id"
+        and "team" in entity.labels
+        and entity.labels["team"] == "matchmaking"
+    )
+
+
+@pytest.mark.parametrize(
+    "test_registry", [lazy_fixture("local_registry")],
+)
+def test_apply_feature_view_success(test_registry):
+    # Create Feature Views
+    batch_source = FileSource(
+        file_format=ParquetFormat(),
+        file_url="file://feast/*",
+        event_timestamp_column="ts_col",
+        created_timestamp_column="timestamp",
+        date_partition_column="date_partition_col",
+    )
+
+    fv1 = FeatureView(
+        name="my_feature_view_1",
+        features=[
+            Feature(name="fs1_my_feature_1", dtype=ValueType.INT64),
+            Feature(name="fs1_my_feature_2", dtype=ValueType.STRING),
+            Feature(name="fs1_my_feature_3", dtype=ValueType.STRING_LIST),
+            Feature(name="fs1_my_feature_4", dtype=ValueType.BYTES_LIST),
+        ],
+        entities=["fs1_my_entity_1"],
+        tags={"team": "matchmaking"},
+        input=batch_source,
+        ttl=timedelta(minutes=5),
+    )
+
+    project: StrictStr = "project"
+
+    # Register Feature View
+    test_registry.apply_feature_view(fv1, project)
+
+    feature_views = test_registry.list_feature_views(project)
+
+    # List Feature Views
+    assert (
+        len(feature_views) == 1
+        and feature_views[0].name == "my_feature_view_1"
+        and feature_views[0].features[0].name == "fs1_my_feature_1"
+        and feature_views[0].features[0].dtype == ValueType.INT64
+        and feature_views[0].features[1].name == "fs1_my_feature_2"
+        and feature_views[0].features[1].dtype == ValueType.STRING
+        and feature_views[0].features[2].name == "fs1_my_feature_3"
+        and feature_views[0].features[2].dtype == ValueType.STRING_LIST
+        and feature_views[0].features[3].name == "fs1_my_feature_4"
+        and feature_views[0].features[3].dtype == ValueType.BYTES_LIST
+        and feature_views[0].entities[0] == "fs1_my_entity_1"
+    )
```

Comment:
I'm worried that this is flaky because it depends on the ordering of things in the features views/features. Can we do something that doesn't rely on indexing into the lists?

Reply:
Good point. I pulled this pattern from `test_feature_store.py`. One thought is that maybe this indexing is deliberate, in order to test that the features were in fact listed in the correct order?

Reply:
Hmmm. I am not sure if we guarantee any ordering in the responses (but happy to address that in a later diff)